server/api/orderitem.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const Sequelize = require('sequelize');
+const router = express.Router();
+const { OrderItem } = require('../../db');
+
+router.use(express.json());
+
+//create an OrderItem. productId and orderId are required
+router.post('/', (req, res, next) => {
+    const { quantity, pricePaid, productId, orderId } = req.body;
+
+    //return 400 if productId & orderId are both not present
+    if (! productId && orderId) {
+        return res.status(400).send('Invalid request');
+    }
+
+    OrderItem.create({
+        quantity: quantity || 1,
```

Comment:
We don't have to use the 'or' operator here because quantity has a default value of 1 in the model definition, right?

Reply:
Yes, the default value will be applied if you create an OrderItem without specifying anything for the quantity field. However, here if we don't have the OR and the request doesn't have a quantity field, the value of `quantity` in this line will be an empty string. That will fail the DB validation, so I'm setting it as 1 to prevent that.